service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -121,14 +135,9 @@ public ProcessorImpl(DSLContext db) {
 
   public Optional<ApplicantReturn> getApplicant(int userId) {
     try {
-      Applicants result = db.select().from(Tables.APPLICANTS).where(Tables.APPLICANTS.USER_ID.eq(userId))
-          .fetchSingleInto(Applicants.class);
-
-      ApplicantReturn ret = new ApplicantReturn(result.getUserId(), result.getResume(), result.getFileType(),
-          Arrays.copyOf(result.getInterests(), result.getInterests().length, String[].class),
-          result.getPriorInvolvement(), result.getWhyJoin());
-
-      return Optional.of(ret);
+      ApplicantReturn result = db.select().from(Tables.APPLICANTS).where(Tables.APPLICANTS.USER_ID.eq(userId))
+          .fetchSingleInto(ApplicantReturn.class);
```

Comment:
Make sure you understand the difference between fetchSingleInto and fetchOneInto because you use both in this PR